app/lib/slack/DeployReporter.scala

```diff
@@ -29,7 +29,7 @@ object DeployReporter {
         val checkpointsAsSlack = checkpoints.map(c => s"<${c.details.url}|${c.name}>").mkString(", ")
         val json = Json.toJson(
           Message(
-            s"*Deployed to $checkpointsAsSlack: ${pr.title}*\n\n${pr.body}",
+            s"*Deployed to $checkpointsAsSlack: ${pr.title}*\n\n${pr.body.getOrElse("")}",
```

Comment:
Shorter and _maybe_ prettier to do:

```
.mkString
```

rather than...

```
.getOrElse("")}
```

... `mkString` works on the `Option` like it was a sequence: giving the empty string for `None`, making the one-element sequence of a `Some` a single string.